cob_hardware_config/robots/raw-mini/urdf/properties.urdf.xacro

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
```

Comment:
do you want to prepare the `cob_calibration_data` structure?

Reply:
on the robot there is not much to calibrate --> I think it is not needed and will not be needed in the future --> no (if that is ok with you)